index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <!-- Header Section-->
+  <header>
+    <nav class="navbar-top-contener"></nav>
+    <nav class="navbar-contener default-display-flex">
```

Comment:
[OPTIONAL]
- [ ] I suggest removing this `nav` element and applying the styles here directly to the `header` element 🤷‍♂️